src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This kind of configuration is no longer needed on spring-boot you can configure the output file from the properties of the application.